docs/content/latest/secure/audit-logging/audit-logging-ysql.md

```diff
@@ -41,35 +41,38 @@ The goal of the YSQL audit logging is to provide YugabyteDB users with capabilit
 
 ### Step 1. Enable audit logging on YB-TServer
 
-This can be done in one of the following ways.
+This can be done in one of the following ways:
 
 ##### Option A: Using `--ysql_pg_conf` TServer flag
 
-1. Database administrators can leverage `ysql_pg_conf` to set appropriate values for pgAudit configuration.
-2. Eg. `ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice"`
-3. These configuration values are set when the YugabyteDB cluster is created and hence are picked up for all users and for every session.
+Database administrators can leverage `ysql_pg_conf` to set appropriate values for pgAudit configuration.
+
+For example, `ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice"`
+
+These configuration values are set when the YugabyteDB cluster is created and hence are picked up for all users and for every session.
 
 
 ##### Option B: Using YugabyteDB `SET` command
 
-1. An alternative suggestion is to use the YB `SET` command, which essentially changes the run-time configuration parameters.
-2. Eg. `SET pgaudit.log='DDL'`
-3. `SET` only affects the value used by the current session. A detailed description of the set command is illustrated in this [webpage](https://www.postgresql.org/docs/8.3/sql-set.html). 
+An alternative suggestion is to use the YB `SET` command, which essentially changes the run-time configuration parameters.
+
+For example, `SET pgaudit.log='DDL'`
+
+`SET` only affects the value used by the current session. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/11/sql-set.html). 
 
 
 ### Step 2. Load the `pgAudit` extension
```

Comment:
pgAudit or pgaudit? Seeing both used throughout. I think pgaudit?

Reply:
It should be pgAudit in text and pgaudit in code. I fixed it.